android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -129,6 +130,23 @@ protected void onCreate(Bundle savedInstanceState) {
     checkSendCrashReport();
     if (KMManager.getMaySendCrashReport()) {
       SentryAndroid.init(context, options -> {
+        options.setBeforeBreadcrumb((breadcrumb, hint) -> {
+          String NAVIGATION_PATTERN = "^(.*)?(keyboard\\.html#[^-]+)-(.)*$";
```

Comment:
```suggestion
          String NAVIGATION_PATTERN = "^(.*)?(keyboard\\.html#[^-]+)-.*$";
```

It's the same but removes the unnecessary group.

Also, could consider including the digits:

```suggestion
          String NAVIGATION_PATTERN = "^(.*)?(keyboard\\.html#[^-]+-(\d+))(\\+.*)?$";
```

But I'm not sure if there are other patterns we'd need to exclude then?